src/table/body-cell/styles.scss

```diff
@@ -95,17 +99,20 @@ $cell-offset: calc(#{awsui.$space-m} + #{awsui.$space-xs});
 }
 @mixin cell-padding-block($padding) {
   > .body-cell-content {
-    padding-block: calc(#{$padding} - 1 * #{awsui.$border-divider-list-width});
```

Comment:
using small negative margins to allow absolutely positioned focus outlines to slightly expand to the top/bottom past cell content boundaries